xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h

```diff
@@ -114,6 +114,7 @@ class CDVDInputStream
     virtual double GetTimeStampCorrection() { return 0.0; };
     virtual bool GetState(std::string &xmlstate) = 0;
     virtual bool SetState(const std::string &xmlstate) = 0;
+    virtual bool CanBeSeeked() { return !IsInMenu(); };
```

Comment:
Grammar suggestion, drop the adjective: `CanSeek()`

Reply:
thanks for the review. Changed the method name accordingly.